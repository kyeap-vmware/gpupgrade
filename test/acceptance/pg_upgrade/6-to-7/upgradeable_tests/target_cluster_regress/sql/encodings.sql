-- Copyright (c) 2017-2024 VMware, Inc. or its affiliates
-- SPDX-License-Identifier: Apache-2.0

-- Test to encodings are preserved after upgrade

--------------------------------------------------------------------------------
-- Create and setup upgradeable objects
--------------------------------------------------------------------------------
SELECT datname, encoding, datcollate, datctype
FROM pg_catalog.pg_database
WHERE datname = 'template0';
