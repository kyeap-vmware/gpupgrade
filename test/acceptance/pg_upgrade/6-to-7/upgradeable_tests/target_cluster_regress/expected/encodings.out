-- Copyright (c) 2017-2024 VMware, Inc. or its affiliates
-- SPDX-License-Identifier: Apache-2.0

-- Test to encodings are preserved after upgrade

--------------------------------------------------------------------------------
-- Create and setup upgradeable objects
--------------------------------------------------------------------------------
SELECT datname, encoding, datcollate, datctype FROM pg_catalog.pg_database WHERE datname = 'template0';
 datname   | encoding | datcollate | datctype 
-----------+----------+------------+----------
 template0 | 0        | C          | C        
(1 row)
